modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustServerCodegen.java

```diff
@@ -219,7 +219,7 @@ public void processOpts() {
         if (additionalProperties.containsKey(CodegenConstants.PACKAGE_NAME)) {
             setPackageName((String) additionalProperties.get(CodegenConstants.PACKAGE_NAME));
         } else {
-            setPackageName("swagger_client");
+            setPackageName("openapi_client");
```

Comment:
@bjgill FYI. I've renamed the default value to "openapi_client" (similar to what we've done in other generators)